README.md

```diff
@@ -215,6 +215,37 @@ You can also set specific database connection to be used by adding `--database`
 ```php
 $this->loadLaravelMigrations(['--database' => 'testbench']);
 ```
+
+#### Runing Testing Migrations
+To run migrations that are only used for testing purposes and not part of your package, add the following to your base test class:
+
+```php
+    public function setUp()
+    {
+        parent::setUp();
+
+        $this->loadMigrationsFrom(__DIR__ . '/database/migrations');
+        
+        // and other test setup steps you need to perform
+    }
+
+    /**
+     * @SuppressWarnings(PHPMD.UnusedFormalParameter)
+     */
+    protected function getPackageProviders($app)
+    {
+        return [
+            // your package service provider,
+            Orchestra\Database\ConsoleServiceProvider\ConsoleServiceProvider::class,
+        ];
+    }
+```
+
+##### Notes and Considerations
+- Your migration files has to suite Laravel's convention, e.g. 0000_00_00_000000_create_package_test_tables.php
+- You may choose to put your migrations folder in tests/database/
```

Comment:
`You may choose to put your migrations folder in tests/database/` 

Only testing used directly for testing should go under `tests`. Otherwise it should go to `resources/database` or etc.